geom/geocad/inc/TOCCToStep.h

```diff
@@ -31,6 +31,8 @@ class TOCCToStep {
 
    STEPCAFControl_Writer    fWriter; //the step file pointer
    Handle(TDocStd_Document) fDoc;    //the step document element
+
+   // The following probably shouldn't be data members. 
```

Comment:
If some of the data members could be just local variables, could you do the change to reflect that?

Reply:
Sorry for the delayed response. When I get some time I'll look into this.